analyze.js

```diff
@@ -60,11 +60,14 @@ const run = async () => {
   }
 
   const results = {};
+  results['total'] = 0;
   for (const key in entries) {
     const entry = entries[key];
-
-    results[key] = entry.value / entry.count;
+    const avg = entry.value / entry.count;
+    results[key] = avg;
+    results['total'] += avg;
```

Comment:
instead of adding everything, then subtracting from the total:

```
if (!Number.isNaN(Number(key)) {
  results['queries'] += avg;
}

results[key] = avg;
results['total'] += avg;
```

might be a little cleaner than trying to subtract.

only caveat being that if the query has a "name" instead of a number.